etc/recipes/lsst-product-configs/meta.yaml

```diff
@@ -13,8 +13,8 @@ build:
   string: "0"
 
 source:
-  git_tag: 1.0.0
-  git_url: https://github.com/mjuric/legacy_configs
+  git_tag: master
+  git_url: https://github.com/jhoblitt/legacy_configs.git
```

Comment:
Ugh -- I was sloppy last night.  This changeset should not have been part of the PR.